### PR TITLE
Multilingual & Layout System & Scss

### PR DESCRIPTION
**Pull request by redbeed**
Hello,

first of all i removed the config.js file an renamed to config.example.js, to protect your config.js.
So before you can run the system you must rename the file.

- Now you can set your language (for forecast & angularjs and momentjs) in the config.
 - I Add a new Layout, you can choose between "main" and "icesnow".
- for icensow i use skycons (animated weather icons)
- You can set time based greetings.
- SCSS files for css

### CODE REVIEW

**Comment by evancohen** [comment]
This branch also has conflicts with `dev`, once you fix them I'll merge this in.

**Comment by redbeed** [comment]
Done :-)
Also i add annyang and angular-dynamic-locale into bower.json

File: js/controller.js [hunk 1]
```diff
@@ -117,7 +177,12 @@
             });
 
             // Go back to default view
-            AnnyangService.addCommand('Wake up', defaultView);
+            AnnyangService.addCommand('wake up', function(){
+                defaultView();
+                //$interval.cancel(sleepTimer);
+                //$interval(sleepTimer, 120000);
+                //sleepTimer = undefined;
+            });
```
Comment (evancohen) [inline js/controller.js]: This should be collapsed back into what was there before (more concise):
``` javascript
AnnyangService.addCommand('Wake up', defaultView);
```

File: js/controller.js [hunk 2]
```diff
@@ -100,6 +142,24 @@
...
+            //    && typeof config.sleep_timer.end != 'undefined'){
+            //        console.log(moment().hour() <= config.sleep_timer.start);
+            //        console.log('END: ')
+            //        console.log(moment().hour() > config.sleep_timer.end);
+            //        if(moment().hour() <= config.sleep_timer.start || moment().hour() > config.sleep_timer.end){
+            //            $scope.focus = "sleep";
+            //        }
+            //    }
+            //};
+            //$interval(sleepTimer, 6000);
```
Comment (evancohen) [inline js/controller.js]: Remove these comments

**Comment by evancohen** [comment]
Two things and then I'll merge this in :)

**Comment by redbeed** [comment]
Fixed :+1:

**Comment by evancohen** [comment]
Awesome work dude. Thanks for your contribution!

**Pull request merged**